reference/commands/creator/create.rst

```diff
@@ -68,9 +68,9 @@ created correctly. Check the 'conan test' command to know more about the
                             binary is not built with the current recipe or when
                             missing binary package. --build=[pattern] Build always
                             these packages from source, but never build the
-                            others. Allows multiple --build parameters. 'pattern'
+                            others. Allows multiple --build parameters. "pattern"
                             is a fnmatch file pattern of a package name. Default
-                            behavior: If you don't specify anything, it will be
```

Comment:
Should it be a way to escape it?

Reply:
Backslashes should make it: http://docutils.sourceforge.net/docs/user/rst/quickref.html#escaping

Reply:
For command line help we should use ``.. code-block:: console``. Howeever the highligth in console mode is too plain and does not show the same as bash. I found a similiar issue here: https://github.com/sphinx-doc/sphinx/issues/4098

Apparently, bash code does not allow ' quotes, that is why we are seeing the warning message all the time. 

Also skipping the ' quotes inside a code-block does not make it work

Reply:
Oh, yes, that is true. I am fine with this merge without the quotes.